### PR TITLE
Fix some typos (found by codespell)

### PR DESCRIPTION
**Pull request by stweil**
Signed-off-by: Stefan Weil <sw@weilnetz.de>

### CODE REVIEW

**Comment by cubbimew** [comment]
upstream cpplint.py still has those "supress"'s

**Comment by neilmacintosh** [comment]
Perhaps it would be best to submit the fixes to upstream cpplint.py and then just pull down the latest version into the repo?

File: scripts/python/cpplint.py [hunk 1]
```diff
@@ -746,7 +746,7 @@ def IsErrorSuppressedByNolint(category, linenum):
     category: str, the category of the error.
     linenum: int, the current line number.
   Returns:
-    bool, True iff the error should be suppressed due to a NOLINT comment or
+    bool, True if the error should be suppressed due to a NOLINT comment or
```
Comment (beinhaerter) [inline scripts/python/cpplint.py]: The "iff" might have been on purpose: https://en.wikipedia.org/wiki/If_and_only_if

**Comment by hsutter** [comment]
Editors call: We don't want to fork, could you please submit the fix upstream?

**Pull request closed without merging**